sass/_omni-search.scss

```diff
@@ -0,0 +1,219 @@
+@import "general-mixins";
+@import "media-queries";
+$form-padding: 0.5rem;
+
+
+@mixin omnisearch-style($version) {
+  @if $version == style-1 {
+    @content; // calls the @mga version here...
```

Comment:
`omnisearch-style` is a kind of container that is looking for an argument that has two possible choices. based on that argument, it will pull in the `omnisearch` mixin, plus a bunch of other styles for that particular pattern (@mga's or mine). Previously `omnisearch` was a large mixin w/ lots of rules inside. This made somewhat inflexible if you needed to make changes. I made `omnisearch` a smaller mixin w/ defaults that @mga had set forth, but were not $variables yet. I then created the the `omnisearch-style` mixin w/ the two arguments and the `@content` directive to wrap the whole thing in and added/chained a new `class`(`.nypl-omnisearch.style-2`) . So we get the `omnisearch` and in my case some variable overrides, plus all the other rules via `@content`. This way we reuse `omnisearch` but arent locked into all the other styles that came w/ that page nor do we break those old designs in favor of the new one. It's convoluted but the combo of arguments & `@content` in the wrapping mixin (`omnisearch-style`) keep these two patterns distinct...

make sense?